src/doom/d_main.c

```diff
@@ -1384,6 +1385,11 @@ void D_DoomMain (void)
 	sidemove[1] = sidemove[1]*scale/100;
     }
     
```

Comment:
You should add a docgen comment on the lines before the M_CheckParm() / M_ParmExists() call to describe what the option does. See other calls to M_CheckParm() throughout the code. This is used to autogenerate manpages.